cucumber/api/features/policy_load_errors.feature

```diff
@@ -82,3 +95,10 @@ Feature: Policy loading error messages
       }
     }
     """
+    And there is an audit record matching:
+    """
+      <85>1 * * conjur * policy
+      [auth@43868 user="cucumber:user:admin"][subject@43868]
+      [action@43868 result="failure" operation="add"]
+      [meta sequenceId="4"] Failed loading policy: policy_text is not present
```

Comment:
@egvili  - should all these be updated too? Failed to load policy....

Reply:
You're right. I will change them as well